pyproject.toml

```diff
@@ -23,7 +23,7 @@ keywords = ['codeflare', 'python', 'sdk', 'client', 'batch', 'scale']
 python = "^3.11"
 openshift-client = "1.0.18"
 rich = ">=12.5,<14.0"
-ray = {version = "2.46.0", extras = ["data", "default"]}
```

Comment:
the minor version was / is required here for poetry.lock to get the correct version